libs/sdk-ui-filters/src/RankingFilter/OperatorDropdown/OperatorDropdown.tsx

```diff
@@ -0,0 +1,72 @@
+// (C) 2020 GoodData Corporation
+import React, { useState } from "react";
+import Button from "@gooddata/goodstrap/lib/Button/Button";
+import { RankingFilterOperator } from "@gooddata/sdk-model";
+import cx from "classnames";
+import { IOperatorDropdownItem } from "../types";
+import { OperatorDropdownBody } from "./OperatorDropdownBody";
+import { WrappedComponentProps, injectIntl } from "react-intl";
+
+const operatorItems = [
+    { value: "TOP", translationId: "rankingFilter.top" },
+    { value: "BOTTOM", translationId: "rankingFilter.bottom" },
```

Comment:
Is it possible to take these values from the types of the filter and not repeat them here?